utils/appConfig.ts

```diff
@@ -0,0 +1,22 @@
+class AppConfig {
```

Comment:
@wolfejw86 I placed the app config and database connection in `/utils`, not 100% sure I'm sold on this structure yet